### DISCUSSION TITLE
GIT Error Compiling with GUI WINDOWS 10

### DISCUSSION THREAD

**Discussion by Jake145** [nrc-cnrc/EGSnrc#970] [Questions] [answered]
Hello, I have followed the installation guide for Windows 10 and this video (https://www.youtube.com/watch?v=ZFBYh7diO8w). However, I have trouble compiling with the GUI. Since I just started, I tried to compile the cavity example to see if it compiles without error, but it gives me the following: 

fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
gfortran -cpp -O2 -mtune=native -fPIC    -DCOMPILE_TIME="\"04/03/2023 10:17 W. Europe Standard Time\""  -c -o egsnrc_win3264.o egsnrc_win3264.F
egsnrc_win3264.F:1440:33:

       write(i_log,'(a)') GIT_HASH
                                 1
Error: Symbol 'git_hash' at (1) has no IMPLICIT type
C:\EGSnrc\HEN_HOUSE\makefiles\cpp_makefile:107: recipe for target 'egsnrc_win3264.o' failed
mingw32-make: *** [egsnrc_win3264.o] Error 1

ExitCode = 2

************************* finished *****************************

Thank you.

**Comment by rtownson** [comment, marked as answer]
This is likely a bug in the most recent release, thanks for letting us know. You can install the 2021 release for now, or wait for a fix (hopefully in a day or two). 

https://github.com/nrc-cnrc/EGSnrc/releases/tag/v2021

**Reply by Jake145** [reply]
Thank you! Your suggestion worked, glad to have brought it up!

**Comment by rtownson** [comment]
See PR #972, the changes to [HEN_HOUSE/specs/all_common.spec](https://github.com/nrc-cnrc/EGSnrc/pull/972/files#diff-761d22ed0e286d1953f225081a37c7d63bb29d476ae73dfb1f2ecd460f4829b7)

This will likely be included in the release soon.
